### ISSUE TITLE
Strange note spacing case

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#289]
Here is a strange spacing case where the second whole notes in the first two measures are not being spaced properly:

![screen shot 2016-09-09 at 5 57 00 pm](https://cloud.githubusercontent.com/assets/3487289/18393765/09ac3b76-76b7-11e6-8d33-29b6e829cf39.png)

It is not related to using cut-time for 2/1 meter, nor related to constrained space:

![screen shot 2016-09-09 at 6 18 27 pm](https://cloud.githubusercontent.com/assets/3487289/18394360/e64ce042-76b9-11e6-8581-28542472b8c7.png)

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="3.0.0">
    <meiHead>
        <fileDesc>
            <titleStmt>
                <title xml:lang="fre" type="main">Pour souhaitter</title>
                <respStmt/>
            </titleStmt>
            <pubStmt>
                <date>2016-09-09 17:55:26</date>
            </pubStmt>
        </fileDesc>
        <encodingDesc>
            <projectDesc/>
        </encodingDesc>
    </meiHead>
    <music>
        <body>
            <mdiv>
                <score>
                    <scoreDef xml:id="scoredef-000000212770961">
                        <staffGrp xml:id="m-000000016210774" symbol="bracket">
                            <staffDef xml:id="staffdef-000000205579255" clef.shape="G" clef.line="2" key.sig="0" meter.count="2" meter.unit="1" meter.sym="cut" n="1" label="Superius" label.abbr="S" lines="5" />
                            <staffDef xml:id="staffdef-000000142046838" clef.shape="G" clef.line="2" clef.dis="8" clef.dis.place="below" key.sig="0" meter.count="2" meter.unit="1" meter.sym="cut" n="2" label="Sextus" label.abbr="Sx" lines="5" />
                            <staffDef xml:id="staffdef-000000014645169" clef.shape="G" clef.line="2" clef.dis="8" clef.dis.place="below" key.sig="0" meter.count="2" meter.unit="1" meter.sym="cut" n="3" label="Contratenor" label.abbr="Ct" lines="5" />
                            <staffDef xml:id="staffdef-000000161184413" clef.shape="G" clef.line="2" clef.dis="8" clef.dis.place="below" key.sig="0" meter.count="2" meter.unit="1" meter.sym="cut" n="4" label="Quintus" label.abbr="Q" lines="5" />
                            <staffDef xml:id="staffdef-000000069398072" clef.shape="G" clef.line="2" clef.dis="8" clef.dis.place="below" key.sig="0" meter.count="2" meter.unit="1" meter.sym="cut" n="5" label="Tenor" label.abbr="T" lines="5" />
                            <staffDef xml:id="staffdef-000000158581648" clef.shape="F" clef.line="4" key.sig="0" meter.count="2" meter.unit="1" meter.sym="cut" n="6" label="Bassus" label.abbr="B" lines="5" />
                        </staffGrp>
                    </scoreDef>
                    <section xml:id="section-000000085903048">
                        <measure xml:id="measure-000000138211086" n="1">
                            <staff xml:id="staff-000000139005992" n="1">
                                <layer xml:id="layer-000000140324549" n="1">
                                    <note xml:id="note-L19F6" dur="1" oct="4" pname="d" accid.ges="n" />
                                    <note xml:id="note-L20F6" dur="1" oct="4" pname="f" accid.ges="n" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-000000051825879" n="2">
                                <layer xml:id="layer-000000167637061" n="1">
                                    <mRest xml:id="mrest-000000046895760" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-000000143521558" n="3">
                                <layer xml:id="layer-000000162531043" n="1">
                                    <mRest xml:id="mrest-000000166105447" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-000000009321853" n="4">
                                <layer xml:id="layer-000000125295004" n="1">
                                    <rest xml:id="rest-L19F3" dur="1" />
                                    <note xml:id="note-L20F3" dur="1" oct="3" pname="a" accid.ges="n" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-000000090006340" n="5">
                                <layer xml:id="layer-000000199699929" n="1">
                                    <mRest xml:id="mrest-000000191631992" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-000000003680389" n="6">
                                <layer xml:id="layer-000000046009637" n="1">
                                    <mRest xml:id="mrest-000000144126611" />
                                </layer>
                            </staff>
                        </measure>
                        <measure xml:id="measure-000000209692624" n="2">
                            <staff xml:id="staff-000000154402904" n="1">
                                <layer xml:id="layer-000000067145242" n="1">
                                    <note xml:id="note-L22F6" dur="1" oct="4" pname="e" accid.ges="n" />
                                    <note xml:id="note-L23F6" dur="1" oct="4" pname="d" accid.ges="n" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-000000138431950" n="2">
                                <layer xml:id="layer-000000000317297" n="1">
                                    <mRest xml:id="mrest-000000132755505" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-000000169665513" n="3">
                                <layer xml:id="layer-000000163721716" n="1">
                                    <mRest xml:id="mrest-000000089065152" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-000000049654352" n="4">
                                <layer xml:id="layer-000000182629605" n="1">
                                    <note xml:id="note-L22F3" dur="1" oct="4" pname="c" accid.ges="n" />
                                    <note xml:id="note-L23F3" dur="1" oct="3" pname="b" accid.ges="n" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-000000086686918" n="5">
                                <layer xml:id="layer-000000189170950" n="1">
                                    <mRest xml:id="mrest-000000111033184" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-000000201607518" n="6">
                                <layer xml:id="layer-000000124105716" n="1">
                                    <mRest xml:id="mrest-000000150359068" />
                                </layer>
                            </staff>
                        </measure>
                        <measure xml:id="measure-000000092104371" n="3">
                            <staff xml:id="staff-000000101553157" n="1">
                                <layer xml:id="layer-000000204111982" n="1">
                                    <rest xml:id="rest-L25F6" dur="2" />
                                    <note xml:id="note-L27F6" dur="2" oct="4" pname="a" accid.ges="n" stem.dir="up" />
                                    <note xml:id="note-L29F6" dur="2" oct="4" pname="a" accid.ges="n" stem.dir="up" />
                                    <note xml:id="note-L31F6" dur="2" oct="4" pname="a" accid.ges="n" stem.dir="up" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-000000026967662" n="2">
                                <layer xml:id="layer-000000026095725" n="1">
                                    <mRest xml:id="mrest-000000175146099" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-000000180083599" n="3">
                                <layer xml:id="layer-000000188144187" n="1">
                                    <note xml:id="note-L25F4" dur="4" oct="3" pname="d" accid.ges="n" stem.dir="up" />
                                    <note xml:id="note-L26F4" dur="4" oct="3" pname="e" accid.ges="n" stem.dir="up" />
                                    <note xml:id="note-L27F4" dur="4" oct="3" pname="f" accid.ges="n" stem.dir="up" />
                                    <note xml:id="note-L28F4" dur="4" oct="3" pname="g" accid.ges="n" stem.dir="up" />
                                    <note xml:id="note-L29F4" dur="4" oct="3" pname="a" accid.ges="n" stem.dir="up" />
                                    <note xml:id="note-L30F4" dur="4" oct="3" pname="g" accid.ges="n" stem.dir="up" />
                                    <note xml:id="note-L31F4" dur="4" oct="3" pname="a" accid.ges="n" stem.dir="up" />
                                    <note xml:id="note-L32F4" dur="4" oct="3" pname="b" accid.ges="n" stem.dir="down" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-000000044588264" n="4">
                                <layer xml:id="layer-000000128297623" n="1">
                                    <note xml:id="note-L25F3" dur="1" oct="3" pname="a" accid.ges="n" />
                                    <rest xml:id="rest-L29F3" dur="1" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-000000026131869" n="5">
                                <layer xml:id="layer-000000153629168" n="1">
                                    <mRest xml:id="mrest-000000069893402" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-000000155711529" n="6">
                                <layer xml:id="layer-000000166679419" n="1">
                                    <note xml:id="note-L25F1" dur="1" oct="3" pname="d" accid.ges="n" />
                                    <note xml:id="note-L29F1" dur="1" oct="3" pname="f" accid.ges="n" />
                                </layer>
                            </staff>
                        </measure>
                        <measure xml:id="measure-000000202431657" n="4">
                            <staff xml:id="staff-000000142462638" n="1">
                                <layer xml:id="layer-000000060468116" n="1">
                                    <note xml:id="note-L34F6" dots="1" dur="2" oct="4" pname="g" accid.ges="n" stem.dir="up" />
                                    <note xml:id="note-L37F6" dur="4" oct="4" pname="e" accid.ges="n" stem.dir="up" />
                                    <note xml:id="note-L38F6" dur="4" oct="4" pname="f" accid.ges="n" stem.dir="up" />
                                    <note xml:id="note-L39F6" dur="4" oct="4" pname="g" accid.ges="n" stem.dir="up" />
                                    <note xml:id="note-L40F6" dur="2" oct="4" pname="a" accid.ges="n" stem.dir="up" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-000000064166734" n="2">
                                <layer xml:id="layer-000000068599976" n="1">
                                    <mRest xml:id="mrest-000000012880811" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-000000038100502" n="3">
                                <layer xml:id="layer-000000021489417" n="1">
                                    <note xml:id="note-L34F4" dur="4" oct="4" pname="c" accid.ges="n" stem.dir="down" />
                                    <note xml:id="note-L35F4" dur="4" oct="4" pname="d" accid.ges="n" stem.dir="down" />
                                    <note xml:id="note-L36F4" dur="4" oct="4" pname="e" accid.ges="n" stem.dir="down" />
                                    <note xml:id="note-L37F4" dur="4" oct="4" pname="c" accid.ges="n" stem.dir="down" />
                                    <note xml:id="note-L38F4" dur="1" oct="4" pname="d" accid.ges="n" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-000000040572374" n="4">
                                <layer xml:id="layer-000000049660933" n="1">
                                    <mRest xml:id="mrest-000000208010108" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-000000040666421" n="5">
                                <layer xml:id="layer-000000119405683" n="1">
                                    <mRest xml:id="mrest-000000109530776" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-000000065109062" n="6">
                                <layer xml:id="layer-000000126824845" n="1">
                                    <note xml:id="note-L34F1" dur="1" oct="3" pname="e" accid.ges="n" />
                                    <note xml:id="note-L38F1" dur="1" oct="3" pname="d" accid.ges="n" />
                                </layer>
                            </staff>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>

```

**Comment by lpugin** [comment]
This was already fixed here 7e31193ccafb03fec5525109c051ef7cfcb8d3b8

**Comment by craigsapp** [comment]
Fix looks good:

![screen shot 2016-09-26 at 9 25 06 am](https://cloud.githubusercontent.com/assets/3487289/18842812/29c0404e-83cb-11e6-8ea6-2ff83ae24610.png)
